tests/test_trigger_bisect_jobs.py

```diff
@@ -89,10 +89,23 @@ def test_catch_CalledProcessError(caplog):
 
     exp_err = "Current job 7848818 will fail, because the repositories for the below updates are unavailable"
     error.stderr=exp_err
-    with patch("subprocess.run", return_value=error):
+    comment_process = subprocess.CompletedProcess(
+        args=[], returncode=0,
+        stderr='',
+        stdout=b'doo'
+    )
+    with patch("subprocess.run", side_effect=[error, comment_process]) as mocked:
         with pytest.raises(SystemExit) as e:
             openqa.main(args)
-
+        mocked.assert_has_calls([
+            call([
+            'openqa-cli', 'api', '--header',
+            'User-Agent: openqa-trigger-bisect-jobs (https://github.com/os-autoinst/scripts)',
+            '--host', 'https://openqa.opensuse.org', '-X', 'POST',
+            'jobs/7848818/comments',
+            'text=Current job 7848818 will fail, because the repositories for the below updates are unavailable'
+            ], stdout=-1, stderr=-1)
+        ])
```

Comment:
I recommend to strip this down to only the relevant bits that are necessary for the test

Reply:
I cant. `mocked.assert_has_calls` doesnt pass if the call doesnt match as a whole. And this is good thing